web/src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -65,6 +65,49 @@
               </v-icon>
               {{ item.name }}
               <v-spacer />
+
+              <v-list-item-action>
```

Comment:
This dialog also renders for folders. While we can debate if we want to allow that, currently the API doesn't support deleting a parent folder, so it should only render on assets. Probably you want something like `v-if="!item.folder"`.

Reply:
Done via 3df03301b56f852bd7cd0e9e8bfe5abb1337daf3 and ebabad3ffd85f099e89183a757985373a4c3d1fd.